pkg/net/libp2p/libp2p.go

```diff
@@ -402,17 +402,23 @@ func discoverAndListen(
 		)
 	}
 
+	newConnManager, err := connmgr.NewConnManager(
```

Comment:
```suggestion
	connectionManager, err := connmgr.NewConnManager(
```

Reply:
Done in ea1b7946b462718c356608932b53613ae3d879c5.